src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.iOSmacOS.cs

```diff
@@ -343,9 +356,12 @@ private static IDisposable InnerCreateLayer(UIElement owner, CALayer parent, Lay
 				{
 					acrylicBrush.Subscribe(owner, area, backgroundArea, parent, sublayers, ref insertionIndex, fillMask: null);
 				}
-				else if (background is XamlCompositionBrushBase unsupportedCompositionBrush)
+				else if (background is XamlCompositionBrushBase)
 				{
-					Brush.AssignAndObserveBrush(unsupportedCompositionBrush, color => backgroundLayer.FillColor = color)
+					Action onInvalidateRender = () => backgroundLayer.FillColor = Brush.GetFallbackColor(background);
+					background.InvalidateRender += onInvalidateRender;
+					onInvalidateRender();
+					new DisposableAction(() => background.InvalidateRender -= onInvalidateRender)
```

Comment:
@Youssef1313 `disposables.Add(() => background.InvalidateRender -= onInvalidateRender)` should be available available to use.

Reply:
@jeromelaban It should be the same functionality? or is there any difference?

Martin will be refactoring BorderLayerRenderer anyway.